src/globals/scss/overrides/_spacing.scss

```diff
@@ -14,25 +14,49 @@ $spacing-directions: (
 // .govuk-\!-m-0 {
 //   margin: 0;
 // }
-// 
+//
 // [..]
-// 
+//
 // .govuk-\!-mt-1 {
 //   margin-top: [whatever spacing point 1 is...]
 // }
 
 @mixin generate-spacing-overrides($property, $property-shorthand) {
   // For each point in the spacing scale (defined in settings), create an
   // override that affects all directions...
-  @each $scale-point, $scale-value in $govuk-spacing-scale {
+  @each $scale-point, $scale-map in $new-govuk-spacing-scale {
```

Comment:
Can this function be update to use the `govuk-spacing-responsive` mixin? There's a lot of duplication between the two mixins.

Reply:
Good call! Done.